contracts/test/modules/Offers/V1/Offers.t.sol

```diff
@@ -200,6 +205,20 @@ contract OffersV1Test is DSTest {
         vm.stopPrank();
     }
 
+    function testRevert_CannotUpdateOfferWithPreviousAmount() public {
```

Comment:
I'm open to renaming this (and the other) function name to match whatever naming convention the team prefers. ✌️